client/src/components/reader/ReturnMenuBody.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+import HigherOrder from 'containers/global/HigherOrder';
```

Comment:
Please change to this:
```
import { HigherOrder } from 'containers/global';
```
I'm trying to decouple imports from the file system so that if we move things around in a module, we don't have to refactor anything — the last time I moved stuff around, it was a serious pain to fix all the imports. In general, I'd prefer that imports from containers and components only reference the top level module in each directory.